editor/modes/visual-editor/block-list.js

```diff
@@ -106,9 +101,8 @@ export default connect(
 		selectionEnd: getBlockSelectionEnd( state ),
 	} ),
 	( dispatch ) => ( {
-		clearSelectedBlock: () => dispatch( { type: 'CLEAR_SELECTED_BLOCK' } ),
 		onMultiSelect( { start, end } ) {
 			dispatch( { type: 'MULTI_SELECT', start, end } );
 		},
 	} )
-)( clickOutside( VisualEditorBlockList ) );
```

Comment:
This is complex, and I think I'm not getting all the nuance, so please correct me if I'm wrong here —  it seems like we have two types of selection:

1. selection when you _click_ the block
2. selection when a block is selected via multi select

1 is important for the inspector, as we need to be able to adjust parameters there without deselecting the block. So far so good.

> But multi-selection allows us to delete the selected blocks by hitting "backspace" which has no sense if the selection is not focused (backspace should work on the focused input or text area ...)

I feel very daft here, the behavior in master feels pretty okay with me:

- I select across blocks, press backspace, and the blocks are deleted
- I select ... oh I think I see what you're getting at.

If I click inside a text block and press backspace, I delete text. If I then click on the _border_, the text is unfocused and it seems like the block is focused. Then backspace deletes the block. This is especially important, I think, for the bigger blocks, like image, where you almost always select the _block_.

If I understand you correctly, the big challenge here is that there's confusion between selecting blocks, setting focus on text, and selecting across blocks, correct?

It seems like you're saying selecting a block (clicking on the border), should highlight the block the same way as multi select does, because it's technically almost the same? Let's imagine what would happen if we did that —  would this be desirable or not? Hmm. Thinking.